### PR TITLE
Add Delete Button

### PR DESCRIPTION
**Pull request by Anita77777**
https://github.com/Anita77777/capstone-project/issues/11
Create delete button for card component

### CODE REVIEW

File: src/components/useStore/useStore.js [hunk 1]
```diff
@@ -3,26 +3,31 @@ import { persist } from 'zustand/middleware';
...
+
+		deleteBooks: id => {
+			set(state => {
+				return {
+					newBooks: state.newBooks.filter(books => books.id !== id),
+				};
+			});
+		},
+	}))
 );
```
Comment (doemser) [inline src/components/useStore/useStore.js]: good job! you could call this function "deleteBook" because it always deletes one book.

**Review by doemser — CHANGES_REQUESTED** [review]
Good Work! :)

Try to handle the bookmarked useStates you use in Form.js within your useStore as well. Remember how we checked a todo in our todo-app? Checking a todo is pretty similar to bookmarking it.

**Comment by doemser** [comment]
In Slack you wrote:
"Thanks for the review. The bookmarks are another branch wich were already approved. Should I just apply the changes you requested regarding the bookmarks on this branch?"

I see that there is a branch that was about the bookmark feature and was already merged. The changes I requested are regarding another feature, so you could go for a refactoring of your bookmark functionality in a separate PR to keep working on one feature at once.

As long as you promise to refactor it and change the function naming, I will approve your PR here and wait for your next one where you refactor it :P

**Comment by Anita77777** [comment]
I changed the function name and will immediately work on the refactoring of the bookmark buttons

**Review by doemser — APPROVED** [review]
:)

**Pull request merged**